proper_parenthetics.py

```diff
@@ -0,0 +1,23 @@
+"""Checks for proper parenthesis."""
+
+
+def parenthesis_checker(text):
+    """Parse given text for proper use of parenthesis."""
+    txt_list = list(text)
+    counter = 0
+    test_list = []
+    for char in txt_list:
+        if counter == -1:
+            return -1
+        elif char == '(':
+            test_list.extend(char)
```

Comment:
You don't need to use `extend` to add a one-character string to a list. `append` would have been more appropriate here.